### PR TITLE
Removed unused MockManager class

### PR DESCRIPTION
**Pull request by Carlinho89**

### CODE REVIEW

File: .idea/vcs.xml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project version="4">
+  <component name="VcsDirectoryMappings">
+    <mapping directory="$PROJECT_DIR$" vcs="Git" />
+    <mapping directory="$PROJECT_DIR$" vcs="Git" />
+  </component>
+</project>
+
```
Comment (kordianbruck) [inline .idea/vcs.xml]: Why are you commting these files?!

Comment (Carlinho89) [inline .idea/vcs.xml]: I think it was Android Studio, maybe they aren't in the gitignore?

Comment (kordianbruck) [inline .idea/vcs.xml]: Well yeah, it is an android studio file but don't you guys check the files you commit before commiting them?

Comment (Carlinho89) [inline .idea/vcs.xml]: how can I solve this issue?

Comment (kordianbruck) [inline .idea/vcs.xml]: Forget it, I'll do some cleanup later..

**Pull request merged**